test/util_test.dart

```diff
@@ -52,3 +53,11 @@ void main() {
     });
   });
 }
+
+extension LineX on Line {
```

Comment:
If you only use it in this file, you can drop the `LineX` name.
```suggestion
extension on Line {
```

Reply:
Thanks, I did not even know that 😄

Reply:
Great solution to make this a testing function